### ISSUE TITLE
Is there a way to use the same port to handle both WebTransport connections, and HTTP3 request/response scheme?

### ISSUE THREAD

**Issue by yaziciahmet** [quic-go/webtransport-go#141]
First of all, thanks for all your work, it is truly appreciated. I have a possible ignorant question.Is there a way to expose both the WebTransport connections, and also traditional REST apis using HTTP3, using the same `http3.Server` instance on the same port? So I want to be able to dial `https://example.com/webtransport` and create a webtransport connection. But I also want to be able to send `GET https://example.com/user` request, and get the details of the user. Is it achievable?

**Comment by marten-seemann** [comment]
Yes, indeed that's possible. The `webtransport.Server` embeds an `http3.Server`, which can handle regular HTTP requests.
